tox.ini

```diff
@@ -1,23 +1,24 @@
 [tox]
 envlist =
-  {py27,py34}-dj{18,19,110}
-  {py27,py34,py35,py36}-dj111
-  {py34,py35,py36}-dj20
-  {py35,py36}-djdev
+  {py35,py36}-dj111
+  {py35,py36,py37,py38}-dj20
+  py39-dj{22,30,31,32,40}
```

Comment:
Is there a reason that you test Django 2.0, but not 2.1?

Reply:
Nope, added it in as an additional test.